### ISSUE TITLE
Fetch bug

### ISSUE THREAD

**Issue by guybedford** [bytecodealliance/StarlingMonkey#19]
I tried enabling the fetch path in the smoke test component here - https://github.com/fermyon/StarlingMonkey/blob/main/tests/smoke.js#L135.

But on the `await textP` line I get the following error:

```
stdout [1] :: Log: Error: TypeError: Response body object should not be disturbed or locked.
```

This is currently blocking enabling `fetch` on ComponentizeJS (https://github.com/bytecodealliance/ComponentizeJS/issues/96).

**Comment by jdegoes** [comment]
We have a fix for some issues with fetch here, if anyone needs it urgently:

https://github.com/bytecodealliance/StarlingMonkey/pull/49

**Comment by tschneidereit** [comment]
Fixed by #52
